src/cephadm/cephadm.py

```diff
@@ -2708,36 +2708,14 @@ def get_container_mounts(
 
     assert ident.fsid
     assert ident.daemon_id
+    # Ceph daemon types are special cased here beacause of the no_config
+    # option which JJM thinks is *only* used by cephadm shell
```

Comment:
I think that's right. Every other time you'd want a ceph container running I can think of it would be for an actual daemon and would need the ceph conf to be present.

Reply:
Again, in the spirit of not changing things that don't immediately need to be changed I left the behavior as-is rather than working on cephadm shell to alter how this property is determined, and just left a breadcrumb comment instead. :-)